src/App.jsx

```diff
@@ -14,7 +17,19 @@ const App = () => {
   return (
     <div className="app">
       <Header />
-      {!isLoading ? <Home /> : <p>Getting things ready for you...</p>}
+      {!isLoading ? (
```

Comment:
you may want give the user an ability to contemplate at least something while your data is loading. I mean not only Getting ready message. Meaning, think about moving loading logic inside Home page. Or maybe release Footer and Header so that they could click around while data is laoding.